tests/pkg_cmsis-nn/Makefile

```diff
@@ -0,0 +1,16 @@
+BOARD ?= nucleo-l476rg
+include ../Makefile.tests_common
+
+USEPKG += cmsis-nn
+
+BLOBS += input
+
+# Boards that were tested and are known to work
+BOARD_WHITELIST := \
+  b-l475e-iot01a \
+  iotlab-m3 \
+  nucleo-l476rg \
+  stm32f723e-disco
```

Comment:
Just wondering - what's special about these?

Reply:
I had them with me when I wrote this PR, so I'm sure they work :)

Reply:
Sure, but are there any requirements to the MCU for this to work or is it just software?

Reply:
Yes, only with M3 (but this is not explicitly mentioned), M4 and M7. I remember now: that required too much changes in the build system and CPUs to be made more automatic. Something like adding `FEATURES_PROVIDED += arch_cortexm4` (that specific level doesn't exist, yet), etc in CPUs and use `FEATURES_REQUIRED += arch_cortexm` with appropriate `FEATURES_BLACKLIST += arch_cortexm0%`, etc in the package Makefile.dep.

#13174 and #13236 could help but there's work.

Reply:
Better add this to the comment then.
My `same54-xpro` also recognizes a cat, but it's probably a bad idea to start a list of every CortexM(≥3) board here ;)

Reply:
Comment added and also added a couple of new boards (same54-xpro, nrf52dk and nrf52832-mdk) that were tested with success.